synapse/lib/storm.py

```diff
@@ -1084,6 +1084,55 @@ def reqValidPkgdef(pkgdef):
         }
         ''',
     },
+    {
+        'name': 'nodes.import',
+        'descr': 'Import a nodes file hosted at a URL into the cortex. Yields created nodes.',
+        'cmdargs': (
+            ('urls', {'nargs': '*', 'help': 'URL(s) to fetch nodes file from'}),
+            ('--no-ssl-verify', {'default': False, 'action': 'store_true', 'help': 'Ignore SSL certificate validation errors.'}),
+        ),
+        'storm': '''
+        init {
+            $count = (0)
+            function fetchnodes(url, ssl) {
+                $resp = $lib.inet.http.get($url, ssl_verify=$ssl)
+                if ($resp.code = 200) {
+                    $nodes = $lib.list()
+                    for $valu in $resp.msgpack() {
+                        $nodes.append($valu)
+                    }
+                    yield $lib.feed.genr("syn.nodes", $nodes)
+                } else {
+                    $lib.warn("nodes.import got HTTP error code: {code} for {url}", code=$resp.code, url=$url)
+                    $lib.exit()
+                }
+            }
+        }
+
+        $ssl = (not $cmdopts.no_ssl_verify)
+
+        if $node {
+            $count = ($count + 1)
+            if ($node.form() != "inet:url") {
```

Comment:
We could use a switch statement to only handle this node type and treat other nodes as nops; unless this exit was a requirement for implementation?

Reply:
This was just to make it look and somewhat act like the wget command above.